docs/specs/metadata.yml

```diff
@@ -99,3 +99,31 @@ outputs:
   build.manifest:
   build.log: |
     ["warning","redirected-id-conflict","Multiple documents redirected to '/sccm/mdm/understand/hybrid-mobile-device-management' with document id: 'sccm/mdm/index.md', 'sccm/mdm/index2.md'","docfx.yml"]
+---
+# generate document id with depot mapping and directory mapping
+# id = hash({mapped_depot_name}|{mapped_file_in_docset})
+# version_independent_id = hash({mapped_depot_name}|{file_path})
+inputs:
+  docfx.yml: |
+    documentId:
+      siteBasePath: azure
+      depotMappings:
```

Comment:
can we keep the original `_op_documentIdPathDepotMapping` syntax? `file -> path + depot`

Reply:
the original syntax is hard to understand and a little confused. and also it contains a lot of duplications. I prefer to splite them and reverse them

---
In reply to: [203581517](https://github.com/dotnet/docfx/pull/3095#discussion_r203581517) [](ancestors = 203581517)